.github/workflows/CI.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         node-version: [14.x, 16.x, 18.x]
     steps:
       - name: Checkout
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v3
```

Comment:
Maybe we should use only major GitHub Actions versions? I'm skeptical about it when it comes to code dependencies, but this CI is not business-critical.

Reply:
Honestly, I don't know. I don't think there is a common standard in actions versioning and most commonly I see just the major versions being used. I thought about locking this to a specific version but after a few minutes of searching for the latest version tag I just decided to use the major one. At the very least I think `actions/checkout` can be trusted. I also don't think I've ever seen using just major versions causing problems yet. 

Conclusion: I'm torn.

Reply:
Then let's pin it to the exact one for now.